test/benchmark/main.cc

```diff
@@ -37,4 +39,11 @@ int main(int argc, char** argv) {
   benchmark::RunSpecifiedBenchmarks();
 }
 
-bool Envoy::benchmark::skipExpensiveBenchmarks() { return skip_expensive_benchmarks; }
+bool Envoy::benchmark::skipExpensiveBenchmarks() {
+  if (skip_expensive_benchmarks) {
+    ENVOY_LOG_MISC(
+        warn, "Expensive benchmarks are being skipped; see test/README.md for more information");
```

Comment:
I would just log in main before the call to benchmark::RunSpecifiedBenchmarks()

Also, we may want to LOG at critical level so for visibility.

Reply:
Done.